build.gradle

```diff
@@ -183,6 +190,7 @@ jib {
         mainClass = 'io.emeraldpay.dshackle.StarterKt'
         args = []
         ports = ['2448', '2449', '8545']
+        environment = ["SPRING_PROFILES_ACTIVE": "docker"]
```

Comment:
looks like we need to have it like an option, because our providers also use this image

Reply:
Agree, thanks